sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/http/client/DefaultHttpClient.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.clientcore.core.implementation.http.client;
+
+import io.clientcore.core.http.client.HttpClient;
+import io.clientcore.core.http.models.HttpRequest;
+import io.clientcore.core.http.models.Response;
+import io.clientcore.core.instrumentation.logging.ClientLogger;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.time.Duration;
+import java.util.Set;
+
+/**
+ * HttpClient implementation using {@link HttpURLConnection} to send requests and receive responses.
+ */
+public final class DefaultHttpClient implements HttpClient {
+    private static final ClientLogger LOGGER = new ClientLogger(DefaultHttpClient.class);
+
+    private static final String ERROR_MESSAGE = "Usage of DefaultHttpClient is only available when using Java 12 or "
+        + "higher. For support with Java 11 or lower, please including a dependency on io.clientcore:http-okhttp3.";
```

Comment:
```suggestion
        + "higher. For support with Java 11 or lower, include a dependency on io.clientcore:http-okhttp3.";
```

The doc tools have been encouraging me to drop "pleases" etc.